src/rpc/mining.cpp

```diff
@@ -302,7 +302,7 @@ static RPCHelpMan generatetoaddress()
 static RPCHelpMan generateblock()
 {
     return RPCHelpMan{"generateblock",
-        "\nMine a block with a set of ordered transactions immediately to a specified address or descriptor (before the RPC call returns)\n",
```

Comment:
I'm not familiar with RPCHelpMan. Why can we drop the `\n` here and at the two other occurrences?

Reply:
Starting and trailing whitespace is normalized by `RPCHelpMan`. See also the rendered diff: https://github.com/bitcoin/bitcoin/pull/24137#issuecomment-1019976927